cucumber-cpp/BodyMacro.hpp

```diff
@@ -21,7 +21,17 @@
             using myBase::myBase;                                                               \
             void Execute(const std::vector<std::string>& args) override                         \
             {                                                                                   \
-                InvokeWithArg(this, args, &BODY_STRUCT::ExecuteWithArgs);                       \
+                SetUp();                                                                        \
```

Comment:
Possibly make it more explicit that 'base' is expected to implement those functions. i.e. use myBase::Setup();
Or abstract to an interface